easybuild/easyconfigs/f/FlexiBLAS/FlexiBLAS-3.0.4-GCC-11.2.0.eb

```diff
@@ -17,7 +17,6 @@ builddependencies = [
     ('CMake', '3.21.1'),
     ('Python', '3.9.6'),  # required for running the tests
     ('BLIS', '0.8.1'),
-    ('imkl', {'arch=x86_64': '2021.4.0', 'arch=*': False}, '', True),
```

Comment:
Or perhaps only picking it up if it is already built...

Reply:
my idea was to let the imkl easyblock build a flexiblas compatible backend, so it can be slotted in via `FLEXIBLAS_LIBRARY_PATH`. This way there are no dependencies at build time.

Here's how to build a GNU-compatible backend with just imkl:
```
module load imkl
cat $MKLROOT/tools/builder/{blas,cblas,lapack}_example_list > user_example_list
make -f $MKLROOT/tools/builder/makefile libintel64 parallel=gnu IFACE_COMP_PART=gf name=libflexiblas_imkl
```
but for now the commenting out is fine